domino/domino.py

```diff
@@ -243,6 +234,164 @@ def runs_stdout(self, runId):
         # pprint.pformat outputs a string that is ready to be printed
         return pprint.pformat(self._get(url)['stdout'])
 
+    def job_start(self, command, commit_id=None, hardware_tier_name=None,
```

Comment:
Is it possible to have a return type hint here, even if it is a dict? (Nice to have)